dali/kernels/common/join/tensor_join_cpu.h

```diff
@@ -106,7 +81,7 @@ TensorShape<> DetermineShape<false>(span<const TensorShape<>> in_shapes, int axi
  * @tparam new_axis if true, STACK mode is applied.
  */
 template<typename T, bool new_axis = true, int dims = -1>
-struct TensorJoinCpu {
+struct TensorJoinCPU {
```

Comment:
This doesn't comply with our naming rules. [This one](https://google.github.io/styleguide/cppguide.html#General_Naming_Rules) to be specific. I think I mentioned this already in some previous PRs, this should be `TensorJoinCpu`

Reply:
Funny fact:
I went to the source code of Tensorflow, for those being Google's naming rules. I found that they use both `Cpu` and `CPU` :)
Examples: `SplitOpCPUImpl`, `XlaCompiledCpuFunction`

That being said, I'd like to see some consistency in our code base.

Reply:
We've been there already. Maybe it doesn't comply, but it wins the "reality check" - 3000 CPU vs 700 Cpu in our code.